src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,76 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { Oval } from 'react-loader-spinner';
+import styles from './Loader.module.css';
+
+// TODO: some props are shadowed from react-loader-spinner. Improve?
+export type Props = {
```

Comment:
This seems fine because we can also document the props ourselves with comments like you've done. Also, the types for `ariaLabel` or `visible` are straightforward

Reply:
yah, we can always revisit this in case there's a much better way to pluck them from the spinner type directly. I did try to do a type union, but there's some issue with the lib since it doesn't export the interface for `OvalProps` (plus, we don't want to export all the props anyhow... this component is a subset that may change in the future)